Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "0.7.50"
+version = "0.7.51"
```

this doesn't require a version bump as it isn't user visable.
But a bump is permitted

---

Yeah, you are right, it isn't necessary here.
```suggestion
version = "0.7.50"
```